wiki/Contests/Skinning_Contest/2/en.md

```diff
@@ -75,9 +108,20 @@ The Skinning Contest #2 is run by various community members.
 - The file size for your .osk submission must be below 100 MB.
 - Please submit a 16:9 image representing your skin, since this will be used as a cover on the contest page.
 
-[Flag_DE]: /wiki/shared/flag/DE.gif "Germany"
-[Flag_GB]: /wiki/shared/flag/GB.gif "United Kingdom"
-[Flag_NL]: /wiki/shared/flag/NL.gif "Netherlands"
-[Flag_PL]: /wiki/shared/flag/PL.gif "Poland"
-[Flag_RO]: /wiki/shared/flag/RO.gif "Romania"
-[Flag_TR]: /wiki/shared/flag/TR.gif "Turkey"
+[flag_BY]: /wiki/shared/flag/BY.gif "Belarus"
+[flag_BR]: /wiki/shared/flag/BR.gif "Brazil"
+[flag_CA]: /wiki/shared/flag/CA.gif "Canada"
+[flag_FR]: /wiki/shared/flag/FR.gif "France"
+[flag_DE]: /wiki/shared/flag/DE.gif "Germany"
+[flag_ID]: /wiki/shared/flag/ID.gif "Indonesia"
+[flag_JP]: /wiki/shared/flag/JP.gif "Japan"
+[flag_MX]: /wiki/shared/flag/MX.gif "Mexico"
+[flag_MY]: /wiki/shared/flag/MY.gif "Malaysia"
+[flag_NL]: /wiki/shared/flag/NL.gif "Netherlands"
+[flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
+[flag_PL]: /wiki/shared/flag/PL.gif "Poland"
+[flag_RO]: /wiki/shared/flag/RO.gif "Romania"
+[flag_TR]: /wiki/shared/flag/TR.gif "Turkey"
+[flag_GB]: /wiki/shared/flag/GB.gif "United Kingdom"
+[flag_US]: /wiki/shared/flag/US.gif "United States"
+[flag_VN]: /wiki/shared/flag/VN.gif "Vietnam"
```

Comment:
These should be ordered alphabetically.

```suggestion
[flag_BR]: /wiki/shared/flag/BR.gif "Brazil"
[flag_BY]: /wiki/shared/flag/BY.gif "Belarus"
[flag_CA]: /wiki/shared/flag/CA.gif "Canada"
[flag_DE]: /wiki/shared/flag/DE.gif "Germany"
[flag_FR]: /wiki/shared/flag/FR.gif "France"
[flag_GB]: /wiki/shared/flag/GB.gif "United Kingdom"
[flag_ID]: /wiki/shared/flag/ID.gif "Indonesia"
[flag_JP]: /wiki/shared/flag/JP.gif "Japan"
[flag_MX]: /wiki/shared/flag/MX.gif "Mexico"
[flag_MY]: /wiki/shared/flag/MY.gif "Malaysia"
[flag_NL]: /wiki/shared/flag/NL.gif "Netherlands"
[flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
[flag_PL]: /wiki/shared/flag/PL.gif "Poland"
[flag_RO]: /wiki/shared/flag/RO.gif "Romania"
[flag_TR]: /wiki/shared/flag/TR.gif "Turkey"
[flag_US]: /wiki/shared/flag/US.gif "United States"
[flag_VN]: /wiki/shared/flag/VN.gif "Vietnam"
```

Reply:
Oh they should be ordered by their code, i did it by their name 🙃